src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -471,4 +472,56 @@ private function createQueryBuilderBetweenDates(\DateTime $from, \DateTime $to,
 
         return $queryBuilder;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function findOneByChannelAndCustomer(ChannelInterface $channel, CustomerInterface $customer)
```

Comment:
No it's not https://github.com/Sylius/Sylius/pull/5197/files#diff-b3c4fa070df7e585e4d320386f45b379R517, It finds only carts 😝